azurelinuxagent/ga/collect_logs.py

```diff
@@ -39,6 +39,33 @@ def get_collect_logs_handler():
     return CollectLogsHandler()
 
 
+def is_log_collection_allowed():
+    # There are three conditions that need to be met in order to allow periodic log collection:
+    # 1) It should be enabled in the configuration.
+    # 2) The system must be using systemd to manage services. Needed for resource limiting of the log collection.
+    # 3) The python version must be greater than 2.6 in order to support the ZipFile library used when collecting.
+    conf_enabled = conf.get_collect_logs()
+    systemd_present = os.path.exists("/run/systemd/system/")
```

Comment:
Could this systemd path ever be distro dependent?

Reply:
Not from our knowledge, but let me make it a constant.